packages/liquidation-sdk-viem/src/preLiquidation/helpers.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  type AccrualPosition,
+  MathLib,
+  ORACLE_PRICE_SCALE,
+  SharesMath,
+} from "@morpho-org/blue-sdk";
+import type { PreLiquidation } from "./types";
+
+export function getSeizabeCollateral(
+  position: AccrualPosition,
+  preLiquidation: PreLiquidation,
+) {
+  const preLiquidationParams = preLiquidation.preLiquidationParams;
+  const lltv = preLiquidationParams.preLltv;
+  const preLltv = preLiquidationParams.preLltv;
```

Comment:
`lltv` and `preLltv` both use `preLiquidationParams.preLltv`; is this expected ?

Reply:
No, nice catch